src/reflect/scala/reflect/io/ZipArchive.scala

```diff
@@ -240,11 +240,11 @@ final class FileZipArchive(file: JFile, release: Option[String]) extends ZipArch
     val root = new DirEntry(RootEntry)
     dirs.put(RootEntry, root)
     val zipFile = openZipFile()
-    val enum    = zipFile.entries()
+    val `enum`  = zipFile.entries()
```

Comment:
honestly, I think `entries` would be a better and more descriptive name

```suggestion
    val entries  = zipFile.entries()
```